string name_object
bool all
---
bool removed